package.json

```diff
@@ -8,6 +8,7 @@
   "types": "dist/lib/index.d.ts",
   "dependencies": {
     "@transmute/did-key-secp256k1": "^0.2.1-unstable.35",
+    "@transmute/did-key-ed25519": "^0.2.1-unstable.35",
```

Comment:
Is it possible to anchor these against a stable branch so that the `^` doesn't affect the input.

Reply:
Would love to pull in better versions of dependencies, just need to tweak the parts of the code that touch the interfaces exposed by those versions.

Reply:
There are [no recent stable releases](https://www.npmjs.com/package/@transmute/did-key-ed25519), so I pinned these dependencies for now to avoid drift: https://github.com/decentralized-identity/ion-sdk/pull/20/commits/ae0afb8165fc3319448ce293181ae472d0d37085

@OR13 What is the latest version that is safe to use?

Reply:
Definitely a good idea to pin them.

current version of did key is: https://github.com/transmute-industries/did-key.js/blob/main/packages/did-key-ed25519/package.json#L5

but you can see the latest version for the key pair class here:
https://github.com/transmute-industries/verifiable-data/blob/main/packages/ed25519-key-pair/package.json#L5

The goal should always be less dependencies... IMO, it would be better for ION SDK to use the key pair classes instead of the did key wrappers... even though the wrappers are tiny... for the most part the interfaces they expose are the same...